### DISCUSSION TITLE
multiprocessing Rerun

### DISCUSSION THREAD

**Discussion by tom-xingshuo** [fastapi/fastapi#6444] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import uvicorn
import multiprocessing
from fastapi import FastAPI
app = FastAPI()


def demo():
    a = [1]
print('几次')


if __name__ == '__main__':
    process2 = multiprocessing.Process(target=demo)
    process2.start()
    uvicorn.run(app)
# uvicorn main:app --reload
# uvicorn main:app --host 0.0.0.0 --port 8000 --reload
```

### Description

After adding multiprocess code to main, it will run multiple times.  For example, if I add a process to main, this code will run twice.  May I ask why?  Can you help me solve this problem?Thanks！

### Operating System

Windows

### FastAPI Version

0.68.0

### Python Version

3.7.10

**Comment by raphaelauv** [comment]
That is not a fastAPI issue , you can very probably close this issue , thank you
